packages/react-native/src/theme/types.ts

```diff
@@ -43,8 +43,8 @@ export type Components<Output> = Record<string, object> & {
   textField?: ComponentTheme<TextFieldStyles, Output>;
 };
 
-export type ColorMode = 'light' | 'dark' | 'system';
-type Override = Omit<Theme, 'overrides'> & {
+export type ColorMode = 'light' | 'dark' | 'system' | null;
```

Comment:
When would this be null? we should default to 'system', not allow null imho.

Reply:
If they pass the return of `useColorScheme` which can be null I guess? That is why I had to add it in

Reply:
https://reactnative.dev/docs/usecolorscheme